tool/ci_functions.sh

```diff
@@ -0,0 +1,27 @@
+# -*- BASH -*-
+# Manage functions used on a CI.
+# Run `. tool/ci_functions.sh` to use it.
+
+# Create options with patterns `-n !/name1/ -n !/name2/ ..` to exclude the test
+# method names by the method names `name1 name2 ..`.
+# See `ruby tool/test/runner.rb --help` `-n` option.
+function ci_to_excluded_test_opts {
+  local tests_str="${1}"
+  ruby <<EOF
+    skipped_tests = "${tests_str}".split.map { |test| "-n \!/#{test}\$$/" }
```

Comment:
The excluded pattern is the backward matching `-n !/method$/` rather than `-n !/^Foo#method/$`, because the `-n !/^Foo#method/$` does not work in my understanding related to this ticket https://bugs.ruby-lang.org/issues/16936 .

Reply:
Self review: Add the comment why the matching is the backward matching `!/name$/` rather than the perfect matching `!/^name$/` referring to the ticket URL above.